bodhi/server/templates/update.html

```diff
@@ -689,6 +689,15 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              <div class="p-b-1">
+                <div>
+                  <strong>CI Status</strong>
```

Comment:
Ah, ok.  I miscommunicated my request:

- Can you have the ``<strong>CI Status</strong>`` be a link to a static wiki page explaining the CI project.
- Can you have the ``${self.util.ci_status2html(update.ci_status) | n}`` label be a link to the actual results for *this* job.